setup.py

```diff
@@ -33,7 +33,7 @@
     keywords=('robotframework testing '
               'test automation testautomation '
               'atdd bdd faker'),  # arbitrary keywords
-    install_requires=['fake-factory', 'robotframework', 'wrapt'],
+    install_requires=['fake-factory==0.7.4', 'robotframework', 'wrapt'],
```

Comment:
i think we should just swap in `faker` - https://pypi.python.org/pypi/Faker

also, i'd rather not pin the dep to a specific version. it's friendlier to leave it unpinned and let developers who are installing this specify a specific version if they choose to.